unit_tests/filter_catchments_and_add_attributes_unittests.py

```diff
@@ -26,9 +32,13 @@ class test_filter_catchments_and_add_attributes(unittest.TestCase):
     def setUpClass(self):
 
         warnings.simplefilter('ignore')
+        try:
```

Comment:
This `try... except FileNotFoundError` block is now in every unit test. Do you think it would be possible/prudent to build the file existence check into the `get_params_filename` method?

Reply:
Actually, I just saw that you already have the file existence check built in. In that case, wouldn't these `try` blocks be redundant?

Reply:
Ok. I removed all of the try catches from each of the unittest files. I would like the exception fall out for now. Later, I will put a "unit_test_all" file that will run each of them in one command. Changing the try catch won't hurt each of the unit tests being run one at a time, but in the "master" , I can catch it more gracefully and ensure that one unit test can fail but other can continue on.  Ready for review again.